tasks/Task.hpp

```diff
@@ -33,6 +33,8 @@ tasks/Task.cpp, and will be put in the genset_whisperpower_ddc namespace.
 
         std::unique_ptr<VariableSpeedMaster> m_driver;
 
+        bool m_running = false;
```

Comment:
Because there is a manual start button, we should initialize m_running on start, or the software might shut down a running generator on reboot.